pkg/cmd/cli/cmd/login.go

```diff
@@ -142,11 +142,15 @@ func (o *LoginOptions) Complete(f *osclientcmd.Factory, cmd *cobra.Command, args
 		o.StartingKubeConfig = kclientcmdapi.NewConfig()
 	}
 
+	addr := flagtypes.Addr{Value: "localhost:8443", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: false}.Default()
```

Comment:
Notice I changed to `AllowPrefix: false` so that we get rid on `/` or whatever path is provided, and only use the scheme/host/port.